terraform/spec/dependabot/terraform/file_parser_spec.rb

```diff
@@ -9,11 +9,33 @@
 RSpec.describe Dependabot::Terraform::FileParser do
   it_behaves_like "a dependency file parser"
 
-  subject(:parser) { described_class.new(dependency_files: files, source: source) }
+  subject(:parser) do
+    described_class.new(
+      dependency_files: files,
+      source: source,
+      options: {
+        terraform_hcl2: PackageManagerHelper.use_terraform_hcl2?
+      }
+    )
+  end
 
   let(:files) { [] }
   let(:source) { Dependabot::Source.new(provider: "github", repo: "gocardless/bump", directory: "/") }
 
+  # V1 output:                             | V2 output:
+  # =======================================|==================================
+  # {                                      | {
+  #   "module": [                          |   "module": {
+  #     {                                  |     "consul": [
+  #       "consul": [                      |       {
+  #         {                              |         "source": "consul/aws",
+  #           "source": "consul/aws",      |         "version": "0.1.0"
+  #           "version": "0.1.0"           |       }
+  #         }                              |     ]
+  #       ]                                |   }
+  #     }                                  | }
+  #   ]                                    |
+  # }                                      |
```

Comment:
Going to remove this comment for now, as I found it slightly confusing, since this is not the output for the `parse` method, it's the difference between a part of the two binaries.